en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md

```diff
@@ -174,138 +183,127 @@ To retest the API, follow the steps given below:
+3. Click the **Web Application** card.
+4. To create the web application, specify the following values for each of the fields:
 
-Now your application has subscribed to the **Reading List Service** REST API.
+    | **Field**             | **Value**               |
+    |-----------------------|-------------------------|
+    | **Component Name**    | `Reading List Web App`  |
+    | **Description**       | `Front-end application for the reading list service` |
+    | **Buildpack**         | Select **React** because the sample front-end application is a React application built with Vite|
+    | **Project Directory** | **`reading-list-app/reading-list-front-end-with-managed-auth`** |
+    | **Build Command**     | **`npm install && npm run build`**            |
+    | **Build Path**        | **`dist`**                                    |
+    | **Node Version**      | **`18`**                                      |
 
-### Step 5.3: Deploy a web application and invoke the REST API
+5. Click **Create**. This initializes the component with the implementation from your GitHub repository and takes you to the **Overview** page of the component. 
 
-At the moment, any user can invoke the **Reading List Service** REST API via the **readingListApp** application (i.e., using its token) and update the reading list. However, if a user sends a request to retrieve the reading list, the response will also show entries by other users. To allow multiple users to use the application and maintain personal reading lists, you need a front-end application that allows each user to sign in with a unique user ID.
+### Step 6.2: Create a connection to the deployed service
 
-Let's deploy a front-end application to consume the API. This application is designed to personalize the reading lists based on the user ID that it obtains from its identity provider.
+To establish a connection between the web application you created and the deployed service, follow the steps given below:
 
-#### Step 5.3.1: Create a web application component
+1. In the left navigation menu, click **Dependencies** and then click **Connections**.
+2. Click **+ Create**.
+3. In the **Create Connection** pane, click `Reading List Service-Readinglist`.`
+4. Specify values as follows for each of the fields:
 
-{% include "create-web-application-qsg.md" %}
+    | **Field**        | **Value**                  |
+    |------------------|----------------------------|
+    | **Name**         | `Reading List Connection`  |
+    | **Description**  | `Connection between the web application and the reading list service`|
+    
+5. Click **Next**. This displays the service URL of the connection for
+each environment the service is deployed in. In this guide, you will see the service URL for the Development environment.
+6. Click **Finish**. This opens the detailed view of the connection you created. You can copy the Service URL displayed here to use when you configure the web application before deploying it.
```

Comment:
Shall we put copying the serviceURL as a separate step? It. can be missed otehrwise.

Reply:
+1
Done